homeassistant/components/bring/coordinator.py

```diff
@@ -0,0 +1,75 @@
+"""DataUpdateCoordinator for the Bring! integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+
+from python_bring_api.bring import Bring
+from python_bring_api.exceptions import BringParseException, BringRequestException
+from python_bring_api.types import BringItemsResponse, BringList
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class BringData(BringList):
+    """Coordinator data class."""
+
+    items: list[BringItemsResponse]
+
+
+class BringDataUpdateCoordinator(DataUpdateCoordinator[list[BringData]]):
```

Comment:
So we are returning a list of lists?

Reply:
Yes, you can have multiple shopping lists (first list) with a list of items (second list).